kyuubi-server/src/main/scala/org/apache/kyuubi/operation/BatchJobSubmission.scala

```diff
@@ -182,7 +182,7 @@ class BatchJobSubmission(
     OperationLog.removeCurrentOperationLog()
   }
 
-  override protected def runInternal(): Unit = session.handleSessionException {
+  override protected def runKyuubiOperationInternal(): Unit = {
```

Comment:
instead of introducing a new method, how about wrapping it at the caller side with `session.handleSessionException`

Reply:
addressed. please take a look

Reply:
```
session.asInstance[KyuubiSession].handleSessionException
```